src/extrinsic/contract.rs

```diff
@@ -54,12 +57,16 @@ pub type ContractInstantiateWithCodeXt = UncheckedExtrinsicV4<ContractInstantiat
 pub type ContractCallXt = UncheckedExtrinsicV4<ContractCallFn>;
 
 #[cfg(feature = "std")]
-impl<P, Client> Api<P, Client>
+impl<P, Client, Params, Tip> Api<P, Client, Params>
 where
     P: Pair,
     MultiSignature: From<P::Signature>,
     MultiSigner: From<P::Public>,
     Client: RpcClient,
+    Params: ExtrinsicParams,
+    Params: ExtrinsicParams<OtherParams = BaseExtrinsicParamsBuilder<Tip>>,
```

Comment:
I am confused, why is this line here? 

`Params: ExtrinsicParams` should be enough, no?

Reply:
Ah ok, I see this is needed, but you can remove the `Params: ExtrinsicParams` because `Params: ExtrinsicParams<OtherParams = BaseExtrinsicParamsBuilder<Tip>` implies the `Params: ExtrinsicParams`.